fblldbbase.py

```diff
@@ -63,7 +63,7 @@ def evaluateIntegerExpression(expression, printErrors=True):
     output = output[2:]
   elif output.startswith('\\'): # Or as \0 (Dec)
     output = output[1:]
-  return int(output, 16)
+  return int(output, 0)
```

Comment:
Because the stripped prefix indicates the base of the literal, it seems like it would make more sense to preserve that information rather than asking `int()` to attempt to automatically detect.

For example, if the output were `\x20` (decimal 32) and the prefix were stripped, `int()` would assume `20` was in base 10.

Suggestion:
```py
if output.startswith('\\x'): # Booleans may display as \x01 (Hex)
    output = output[2:]
    base = 16
elif output.startswith('\\'): # Or as \0 (Dec)
    output = output[1:]
    base = 10
else:
    base = 10
  return int(output, base)
```

Three additional comments, from my own unfamiliarity:
1. Finally, the `0` prefix is usually used to indicate octal, not decimal. Is that different here?
2. Because the expression given to LLDB is wrapped in `(int)(…)` above, doesn't LLDB *always* return that value in decimal? It isn't attempting to render a pointer type.
3. To accommodate cases of e.g. a `ptrdiff_t` which may be longer than an int, should the above use `(long long)` rather than `(int)`?